src/main/java/edu/ucsb/cs156/example/services/UCSBCurriculumService.java

```diff
@@ -0,0 +1,58 @@
+package edu.ucsb.cs156.example.services;
+
+import java.util.Arrays;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import edu.ucsb.cs156.example.documents.CourseSearchParams;
+
+/**
+ * Service object that wraps the UCSB Academic Curriculum API
+ */
+@Service
+public class UCSBCurriculumService  {
+
+    private Logger logger = LoggerFactory.getLogger(UCSBCurriculumService.class);

```

Comment:
Can we use the `@Slf4j` annotation here instead of manually creating a logger?

Reply:
Fixed with new commit